virttest/utils_sriov.py

```diff
@@ -62,6 +62,30 @@ def set_vf(pci_addr, vf_no=4, session=None):
     return not s
 
 
+def get_vf_pci_id(pf_pci, vf_index=0, session=None):
+    """
+    Get pci_id of VF
+
+    :param pf_pci: The pci id of PF
+    :param vf_index: VF's index
+    :param session: The session object to the host
+    :return: VF's pci id
+    """
+    cmd = "ls /sys/bus/pci/devices/{}/virtfn{}/net".format(pf_pci, vf_index)
```

Comment:
Do we need confirm with sebistian whether the folder is the same on s390x  since it sounds like there are differences in different arch?

Reply:
There is no sriov job for s390x arch, @smitterl  can you please help us confirm if we need to run sriov related test on s390x arch? Thanks!

Reply:
@chunfuwen @Yingshun That's right on s390x we don't test sr-iov.

Reply:
Thank you @smitterl !